AppLovin MAX Demo App - ObjC/AppLovin MAX Demo App - ObjC/AppLovin/MRECs/Zone Integration/ALDemoMrecZoneViewController.m

```diff
@@ -0,0 +1,127 @@
+//
+//  ALDemoMrecZoneViewController.m
+//  AppLovin MAX Demo App - ObjC
+//
+//  Created by Nana Amoah on 11/23/21.
+//  Copyright © 2021 AppLovin Corporation. All rights reserved.
+//
+
+#import "ALDemoMrecZoneViewController.h"
+#import <AppLovinSDK/AppLovinSDK.h>
+
+@interface ALDemoMrecZoneViewController() <ALAdLoadDelegate, ALAdDisplayDelegate, ALAdViewEventDelegate>
+@property (nonatomic, strong) ALAdView *adView;
+@property (nonatomic,   weak) IBOutlet UIBarButtonItem *loadButton;
+@end
+
+@implementation ALDemoMrecZoneViewController
+
+#pragma mark - View Lifecycle
+
+- (void)viewDidLoad
+{
+    [super viewDidLoad];
+    
+    // Create the MREC view
+    self.adView = [[ALAdView alloc] initWithSize: ALAdSize.mrec zoneIdentifier: @"YOUR_ZONE_ID"];
```

Comment:
We can actually omit the zones sample since everyone's moving towards bidding anyways.